### PR TITLE
Update circle.yml file to circle version 2

### PR DESCRIPTION
**Pull request by andybradshaw**

### CODE REVIEW

**Comment by uschi2000** [comment]
needs to fix creds, spoke offline

File: circle.yml [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
-  artifacts:
-    - build/libs
-
-test:
-  override:
-    - ./gradlew --parallel -i build
-
-deployment:
-  release:
-    tag: /[0-9]+(\.[0-9]+)+((-(beta|rc)[0-9]{1,2})(\.[0-9])?)?/
```
Comment (uschi2000) [inline circle.yml:23]: need to ensure that this logic is carried over.

Comment (uschi2000) [reply circle.yml:23]: it's fine to do this in gradle rather than circle

File: circle.yml [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
-    commands:
-      - ./gradlew -i bintrayUpload
-      - ./gradlew -i -Dgradle.publish.key=$GRADLE_KEY -Dgradle.publish.secret=$GRADLE_SECRET publishPlugins
+version: 2
+executorType: machine
+stages:
+  build:
+    workDir: ~/gradle-docker
```
Comment (uschi2000) [inline circle.yml:5]: this is not the default? i'm surprised.

**Comment by uschi2000** [comment]
Any idea why the test results XML doesn't show up in Circle?

**Review by uschi2000 — APPROVED** [review]

**Comment by andybradshaw** [comment]
Not sure... looks like this happened on Mark's Circle 2.0 PR as well. Will play around with it on a branch and see what's up.

Edit: This is fixed by #113

**Pull request merged**